Automattic/wp-calypso
# Taxonomy Manager: Add Error Handling for Crashes

Related to #88590

## Proposed Changes

Taxonomies currently aren't working on Simple sites. I'll take a look at the endpoint to see why this might be happening, but in the meantime, let's add some error handling so that Calypso at least doesn't become unusable for the users whom this affects.

## Testing Instructions

* Go to `/settings/taxonomies`
* Select a site and update/create a taxonomy
* If there's an error, confirm that the dialog now closes and you can still continue seeing Calypso - you should also see a notice informing you of this error.
* If there's no error, confirm that you now see a success notice too.

<img width="1290" alt="Screenshot 2024-03-23 at 12 32 04" src="https://github.com/Automattic/wp-calypso/assets/43215253/33daad79-7a6d-4d63-ac5b-0bc78f6b8e85">

cc @flootr, @tyxla, @vindl (though I recognise none of you have touched this in quite a while!)
